include.js

```diff
@@ -5961,11 +5961,19 @@ let cards = [
     githubLink: 'https://github.com/JinalPatel17'
   },
   {
+<<<<<<< HEAD
```

Comment:
Hello @samialmaradni97 , this indicates Esinnation is already in the remote repository in the `master` branch
```
======
`code snippet`
>>>>>> master
```
The merge conflict arised because in the HEAD of your last commit, you are attempting to add your contributions but your branch doesn't have changes merged from Esination.

In order to fix this in the web UI, you simply copy whatever is between ======== and the master branch right above your additions and remove <<<<<<< HEAD.